source/common/router/config_impl.cc

```diff
@@ -1036,7 +1036,7 @@ absl::optional<std::string> RouteEntryImplBase::currentUrlPathAfterRewriteWithMa
       return std::string(headers.getPathValue());
     }
 
-    return *std::move(new_path);
+    return path.replace(0, just_path.size(), new_path.value());
```

Comment:
Is this a bug fix?

Reply:
Yes. Currently the url params are truncated instead of appended.

Reply:
Ah, I see. In that case it seems like this is a user-visible change. Is that right? If so, we'll probably need a reloadable feature to control it. (Unless this codepath is alpha?)

Reply:
It is not in use anywhere. This was caught before rollout.